src/main/java/org/folio/consortia/service/impl/SharingInstanceServiceImpl.java

```diff
@@ -80,8 +77,7 @@ public SharingInstance start(UUID consortiumId, SharingInstance sharingInstance)
       }
 
       try (var context = new FolioExecutionContextSetter(prepareContextForTenant(targetTenantId, folioModuleMetadata, folioExecutionContext))) {
-        String source = FOLIO_SOURCE_VALUE.equalsIgnoreCase(inventoryInstance.get("source").asText()) ? CONSORTIUM_FOLIO : CONSORTIUM_MARK;
```

Comment:
I am not sure if it still stands but it should be CONSORTIUM_MARC not CONSORTIUM_MARK.

Reply:
fixed